figures/README.md

```diff
@@ -60,6 +61,7 @@ Biospecimens without a `short_histology` designation are coded as `none` and ass
r>ETMR:![009929](https://placehold.it/150x40/009929/FFFFFF?text=009929) <br>Ganglioglioma:![698c7c](https://placehold.it/150x40/698c7c/FFFFFF?text=698c7c) <br>Germinoma:![39e6c3](https://placehold.it/150x40/39e6c3/FFFFFF?text=39e6c3) <br>Glial-neuronal tumor NOS:![005359](https://placehold.it/150x40/005359/FFFFFF?text=005359) <br>Gliosis:![263233](https://placehold.it/150x40/263233/FFFFFF?text=263233) <br>Hemangioblastoma:![00c2f2](https://placehold.it/150x40/00c2f2/FFFFFF?text=00c2f2) <br>Hemangioma:![40a6ff](https://placehold.it/150x40/40a6ff/FFFFFF?text=40a6ff) <br>HGAT:![406280](https://placehold.it/150x40/406280/FFFFFF?text=406280) <br>Langerhans Cell histiocytosis:![0044ff](https://placehold.it/150x40/0044ff/FFFFFF?text=0044ff) <br>LGAT:![00144d](https://placehold.it/150x40/00144d/FFFFFF?text=00144d) <br>LGMT:![acbbe6](https://placehold.it/150x40/acbbe6/FFFFFF?text=acbbe6) <br>Medulloblastoma:![7373e6](https://placehold.it/150x40/7373e6/FFFFFF?text=7373e6) <br>Meningioma:![3d0099](https://placehold.it/150x40/3d0099/FFFFFF?text=3d0099) <br>MPNST:![c200f2](https://placehold.it/150x40/c200f2/FFFFFF?text=c200f2) <br>Neurofibroma:![917399](https://placehold.it/150x40/917399/FFFFFF?text=917399) <br>none:![f1f1f1](https://placehold.it/150x40/f1f1f1/FFFFFF?text=f1f1f1) <br>Oligodendroglioma:![f279da](https://placehold.it/150x40/f279da/FFFFFF?text=f279da) <br>Other:![cc0052](https://placehold.it/150x40/cc0052/FFFFFF?text=cc0052) <br>Pineoblastoma:![994d6b](https://placehold.it/150x40/994d6b/FFFFFF?text=994d6b) <br>Schwannoma:![4d2636](https://placehold.it/150x40/4d2636/FFFFFF?text=4d2636) <br>Teratoma:![ffbfd9](https://placehold.it/150x40/ffbfd9/FFFFFF?text=ffbfd9) <br>loss:![053061](https://placehold.it/150x40/053061/FFFFFF?text=053061) <br>gain:![2166ac](https://placehold.it/150x40/2166ac/FFFFFF?text=2166ac) <br>Fusion:![4393c3](https://placehold.it/150x40/4393c3/FFFFFF?text=4393c3) <br>Multi_Hit:![92c5de](https://placehold.it/150x40/92c5de/FFFFFF?text=92c5de) <br>Multi_Hit_Fusion:![d1e5f0](https://placehold.it/150x40/d1e5f0/FFFFFF?text=d1e5f0) <br>Missense_Mutation:![f7f7f7](https://placehold.it/150x40/f7f7f7/FFFFFF?text=f7f7f7) <br>Nonsense_Mutation:![fddbc7](https://placehold.it/150x40/fddbc7/FFFFFF?text=fddbc7) <br>In_Frame_Del:![f4a582](https://placehold.it/150x40/f4a582/FFFFFF?text=f4a582) <br>In_Frame_Ins:![d6604d](https://placehold.it/150x40/d6604d/FFFFFF?text=d6604d) <br>Frame_Shift_Del:![b2182b](https://placehold.it/150x40/b2182b/FFFFFF?text=b2182b) <br>Frame_Shift_Ins:![67001f](https://placehold.it/150x40/67001f/FFFFFF?text=67001f) <br>Splice_Site:![f1f1f1](https://placehold.it/150x40/f1f1f1/FFFFFF?text=f1f1f1) | A named vector of hex codes assigned to each `short_histology` and to each `CNV`, `SNV` and `Fusion` category | For plotting an oncoprint figure, this vector incorporates the `histology_color_palette.tsv` and `divergent_color_palette.tsv` to provide hex codes for `short_histology` values as well as `CNVs`, `SNVs`, and `Fusions`
```

Comment:
This does not look like it got updated with the most recent changes to the palette.

Reply:
You are correct, will do that in the upcoming commit.